docs/concepts/materializing-features.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: default
-title: Feature Generation and Materialization
+title: Feature Materialization (also known as feature generation)
 parent: Feathr Concepts
 ---
 
-# Feature Generation and Materialization
+# Feature Materialization (also known as feature generation)
 
-Feature generation (also known as feature materialization) is the process to create features from raw source data into a certain persisted storage in either offline store (for further reuse), or online store (for online inference).
+Feature materialization (also known as feature generation) is the process to create features for a certain entity from raw source data into a certain persisted storage in either offline store (for further reuse), or online store (for online inference).
```

Comment:
I would suggest we remove talking point of offline store in this materizliation doc. 

Publish from offline store to online store via materialization and then using online store to inferrence model servicing is in critial path of the feature engineering pipeline, however publish from offline to offline seems not. We better on hold talking about offline 2 offline until there are some good user cases.

Reply:
Actually we should talk a bit here given it's the source of truth doc. The use case is described in the doc like this:

`This is useful when the feature transformation is compute intensive and features can be re-used. For example, you have a feature that needs more than 24 hours to compute and the feature can be reused by more than one model training pipeline. In this case, you should consider generating features to offline.`

Agree it's not on the critical path, but we should still document it, since there are folks asking for those.